### ISSUE TITLE
R120 BPI Service specification

### ISSUE THREAD

**Issue by atoulme** [eea-oasis/baseline-standard#166]
Requirement R120 introduces the notion of `BPI Service specification`, but does not define it. Is it what is defined in R119?

**Comment by atoulme** [comment]
Furthermore, should it keep tally of all updates or only reference the last update timestamp? The requirement is unclear on that front.

**Comment by Therecanbeonlyone1969** [comment]
@atoulme yes, the BPI service is defined in D24 -> we could make it MUST and say that this is the minimal definition. Thoughts?

And R120, yes, it should include the history of "update" time stamps. Will clarify!

```
[R120]
To ensure a BPI service specification timeline, the BPI service specification MUST include “created” and “updated” timestamps,, and the full history of "updated" timestamps. 
```
Is the above good enough?

**Comment by atoulme** [comment]
Oh ok yes I think that makes sense.
